python/graphstorm/eval/evaluator.py

```diff
@@ -440,15 +515,15 @@ def evaluate(self, val_pred, test_pred, val_labels, test_labels, total_iters): #
         for metric in self.metric:
             # be careful whether > or < it might change per metric.
             if self.metrics_obj.metric_comparator[metric](
-                self._best_val_score[metric], val_score[metric]):
+                    self._best_val_score[metric], val_score[metric]):
```

Comment:
indent

Reply:
This indent follow pylint's indent format because this line is part of the above `if self.metrics...` condition line.